### PR TITLE
USHIFT-1472: Deploy and boot ID in version file to avoid unnecessary restores

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: test/image-blueprints/group2/rhel92-source-fake-next-next-minor.toml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name = "rhel-9.2-microshift-source-fake-next-next-minor"
```
Comment (dhellmann) [inline test/image-blueprints/group2/rhel92-source-fake-next-next-minor.toml:1]: This name is a bit subtle when scanning. Can we use YPLUS2 or some other more distinct thing other than "next next"?

File: test/image-blueprints/group2/rhel92-source-fake-next-minor.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name = "rhel-9.2-microshift-source-fake-next-minor"
-description = "A RHEL 9.2 image with the RPMs built from source from the current PR but with the _version_ set to the next y-stream."
+description = "A RHEL 9.2 image with the RPMs built from source from the current PR but with the _version_ set to the current+2 y-stream."
```
Comment (dhellmann) [inline test/image-blueprints/group2/rhel92-source-fake-next-minor.toml:2]: I think you updated the description in the wrong file here?

File: pkg/admin/prerun/version.go [hunk 3]
```diff
@@ -23,6 +24,23 @@ var (
 // CheckAndUpdateDataVersion checks version compatibility between data and executable,
 // and updates data version
 func CheckAndUpdateDataVersion() error {
+	// Q: Should we use separate version schema for ostree and non-ostree?
```
Comment (dhellmann) [inline pkg/admin/prerun/version.go:27]: What do you have in mind for the different version schema?

**Review by dhellmann — COMMENTED** [review]
One nit about naming, but that's probably not worth holding this up. Remove the hold if you're ready to proceed.

/lgtm
/hold

File: test/image-blueprints/group2/rhel92-source-fake-next-next-minor.toml (hunk 1 above)
Comment (pmtk) [reply test/image-blueprints/group2/rhel92-source-fake-next-next-minor.toml]: Yes, thanks for a suggestion, I had no ideas at the moment of writing :stuck_out_tongue_winking_eye:

File: test/image-blueprints/group2/rhel92-source-fake-next-minor.toml (hunk 2 above)
Comment (pmtk) [reply test/image-blueprints/group2/rhel92-source-fake-next-minor.toml]: Fixed

**Comment by dhellmann** [comment]
The linter looks unhappy. Otherwise, this looks good.

**Pull request merged**